sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/SplittableDoFnTest.java

```diff
@@ -93,8 +96,19 @@ public void process(ProcessContext c) {
     }
   }
 
+  // Using testing options with streaming=true makes it possible to enable UsesSplittableParDo tests
+  // in Dataflow runner, because as of writing, it can run Splittable DoFn only in streaming mode.
+  // This is a no-op for other runners currently (Direct runner doesn't care, and other
+  // runners don't implement SDF at all yet).
```

Comment:
This can (and should be) be configured by the test executions. If we don't have any extant streaming execution, we should have one.